Tests/TuistKitTests/Storyboards/Mocks/MockStoryboardGenerator.swift

```diff
@@ -0,0 +1,33 @@
+import Basic
+import Foundation
+@testable import TuistKit
+
+final class MockStoryboardGenerator: StoryboardGenerating {
+    typealias Storyboard = (path: AbsolutePath, name: String, platform: Platform, product: Product, isLaunchScreen: Bool)
+
+    var generatedStoryboards: [Storyboard] = []
```

Comment:
👍 In mocks / stubs - having an array to store the operations conducted is a great idea! Allows verifying counts as well as content!